src/index.html

```diff
@@ -13,6 +13,46 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="product-image"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card-rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="stars-reviews">Reviews: 5</p>
```

Comment:
Consider making this a part of card 
```suggestion
        <p class="card__rating_reviews">Reviews: 5</p>
```